spyder/app/tests/test_cli_options.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Tests for cli_options.py
+"""
+import pytest
+from spyder.app import cli_options
+
+
+def test_get_options():
+    getopt = cli_options.get_options
+
+    options, args = getopt([])
+    assert not options.new_instance
+    assert not options.reset_to_defaults
+    assert not options.reset_config_files
+    assert not options.optimize
+    assert options.working_directory is None
+    assert not options.hide_console
+    assert not options.show_console
+    assert not options.multithreaded
+    assert not options.profile
+    assert options.window_title is None
+    assert options.open_project is None
+    assert options.files == []
+    assert args == []
+
+    options, args = getopt(['--new-instance'])
+    assert options.new_instance
+
+    options, args = getopt(['--defaults', '--reset'])
+    assert options.reset_to_defaults
+    assert options.reset_config_files
+
+    options, args = getopt(['--optimize', '--workdir', 'test dir'])
+    assert options.optimize
+    assert options.working_directory == 'test dir'
+
+    options, args = getopt('--window-title MyWindow'.split())
+    assert options.window_title == 'MyWindow'
+
+    options, args = getopt('-p myproject test_file.py another_file.py'.split())
+    assert options.open_project == 'myproject'
+    assert options.files == ['test_file.py', 'another_file.py']
+    assert args == ['test_file.py', 'another_file.py']
+
+    with pytest.raises(SystemExit):
+        options, args = getopt(['--version'])
```

Comment:
While reviewing this PR, I noticed that spyder doesn't have a `--version` option 

Maybe we could added it (in other PR), @ccordoba12 ?

Reply:
Yep, sure. Please go for it, when you have some free time ;-)